File: src/PHP/CreationalPatterns/AbstractFactory/Furniture/Products/ClassicCabinet.php
<?php

namespace CreationalPatterns\AbstractFactory\Furniture\Products;

use CreationalPatterns\AbstractFactory\Furniture\Interfaces\IFurniture;

class ClassicCabinet implements IFurniture {
    public function showFurnitureStyle(): void
    {
        echo "I am a classic cabinet\n";
    }
}

File: src/C#/CreationalPatterns/AbstractFactory/Furniture/Products/ClassicCabinet.cs
using Furniture.Interfaces;
using System;
using System.Collections.Generic;
using System.Text;

namespace Furniture.Products
{
    public class ClassicCabinet : IFurniture
    {
        public void ShowFurnitureStyle()
        {
            Console.WriteLine("I am a classic cabinet");
        }
    }
}
